raiden/utils/__init__.py

```diff
@@ -175,3 +176,25 @@ def channel_to_api_dict(channel):
         "balance": channel.contract_balance,
         "state": channel.state
     }
+
+
+def create_file_iff_not_existing(path, return_it, **kwargs):
```

Comment:
Agreed. Kind of liked this function and thought it would be good to have anyway but we can bring it back any time we need.